soc/nxp/imxrt/Kconfig

```diff
@@ -96,6 +97,28 @@ config FLEXSPI_CONFIG_BLOCK_OFFSET
 	  sequence (optional), etc. The boot ROM expects FlexSPI configuration
 	  parameter to be presented in serial nor flash.
 
+config IMAGE_CONTAINER_OFFSET
```

Comment:
is this different from IVT on the other RT parts?

Reply:
Yes， RT1180 is different with other RT platforms.

Reply:
@lucien-nxp do we document the details someplace on this?